### ISSUE TITLE
assimp-4.0.1 problems on mesh alignment

### ISSUE THREAD

**Issue by ruellm** [assimp/assimp#1530]
i have reported this issue in gamedev.net thinking that my implementation is wrong:

The mesh blades are miss aligned from the body mesh.
https://www.gamedev.net/forums/topic/693235-assimp-opengl-other-mesh-position-not-correct/?tab=comments#comment-5361397
(you can find screenshots and the FBX file itselfe in the link)

but then i tried to run other people's Assimp loader code using my own build (lib and DLL i have for assimp 4.0.1).
i used the mesh loader code in http://learnopengl.com/ and got the same problem, which leads me to judge that the problem is not on my loader code but maybe on my build/version.

So i went around and look for 3d viewer based from Assimp and found this one: http://assimp.sourceforge.net/main_viewer.html
and ofcourse the one the assimp viewer that is based on 3.3.1 and included in the windows installer (prebuilt).

both viewer displayed the blade meshes correctly.
so I wonder, why does the Assimp i have loads the model differently compare to the viewer?
Is this a legitimate issue on the latest version?

for reference, you can find the FBX file on my gamedev.net thread and the prebuilt files attached on this report.

[assimp-4.0.1.zip](https://github.com/assimp/assimp/files/1437983/assimp-4.0.1.zip)
Built in Windows 10, Visual Studio 2015.

In summary, 
1. using the build i have based from 4.0.1, the blade mesh is not aligned to body mesh. 
I have used my code loader, 
the loader from ogldev (ogldev.atspace.co.uk) and the mesh loader from https://learnopengl.com/, all of which produces the wrong alignment of the second mesh (the blade).

you can download any of the source code from the above links i provide and use the latest 4.0.1 build or the ones i attached to replicate the issue.

2. using the prebuilt libraries and other viewer, FBX displayed correctly.

maybe there is a problem on latest version of assimp?

**Comment by kimkulling** [comment]
Hm, we need to check. Stay tuned ...

**Comment by luigimux** [comment]
any update on this?

**Comment by kimkulling** [comment]
Should be fixed in 4.1.0, sorry for the missing update. Just reopen if this is still an issue.
